server/src/models/Marker.ts

```diff
@@ -0,0 +1,26 @@
+import { Schema, model } from 'mongoose';
+
+const Marker = new Schema(
+  {
+    longitude: {
+        type: Number,
+        required: true,
+      },
+    latitude: {
+        type: Number,
+        required: true,
+    },
+    timestamp: {
+        type: Date,
+        default: Date.now,
+    },
+    type: {
+        type: String,
+        enum: ['default', 'pothole', 'sbump', 'closure','xwalk'], // Add more types as needed
```

Comment:
These are useful types of markers for now. We can look into adding a car accident type or 'suggestion' type in the future. Not sure if speed bumps or crosswalks are needed.

Reply:
Great idea! I was having a bit of trouble thinking of some basic types, but accidents and suggestions sound good to me. Perhaps maybe other would suffice as well? Could update this in the future anyways. I'll add your suggestions to the "Things to Update" portion.